rpc/koinos_rpc_client.go

```diff
@@ -254,7 +259,7 @@ func (c *KoinosRPCClient) SubmitTransaction(ctx context.Context, ops []*protocol
 	}
 
 	// Create the header
-	header := protocol.TransactionHeader{ChainId: chainID, RcLimit: rcLimit, Nonce: nonceBytes, OperationMerkleRoot: merkleRoot, Payer: address}
+	header := protocol.TransactionHeader{ChainId: chainID, RcLimit: rcLimit, Nonce: nonceBytes, OperationMerkleRoot: merkleRoot, Payer: payer}
```

Comment:
If we have a separate `payer`, then we need to specify the `payee` so the `payee`'s nonce is used. And if `payer == payee`, then `payee` should not be set.